lib/services/web-view-service.ts

```diff
@@ -0,0 +1,78 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+export class WebViewService implements IWebViewService {
+	private static MIN_SUPPORTED_VERSION = "4.0.0";
+	
+	constructor(private $errors: IErrors,
+		private $pluginsService: IPluginsService,
+		private $project: Project.IProject,
+		private $projectConstants: Project.IProjectConstants,
+		private $options: IOptions) {
+			this.$project.ensureCordovaProject(); 
+		}
+		
+	public validateFrameworkVersion(): void {
+		if(this.$project.projectData.FrameworkVersion < WebViewService.MIN_SUPPORTED_VERSION) {
+			this.$errors.failWithoutHelp(`Web view releated commands are supported only for Cordova ${WebViewService.MIN_SUPPORTED_VERSION} or greater.`);
+		}
+	}
+	
+	public get supportedWebViews(): IDictionary<IWebView[]> { 
+		return {
+			'ios': [ 
+				{ name: "Built-in", default: true }, 
+				{ name: "WKWebView", marketplacePluginIdentifier: "com.telerik.plugins.wkwebview" }],
+			'android': [
+				{ name: "Built-in", default: true }, 
+				{ name: "Crosswalk", marketplacePluginIdentifier: "com.telerik.plugins.crosswalk-webview" }]
+		}
+	}
+	
+	public getWebViews(platform: string): IWebView[] {
+		 return this.supportedWebViews[platform.toLowerCase()];	
+	}
+	
+	public getWebViewNames(platform: string): string[] {
+		let webViews = this.getWebViews(platform);
+		return _.map(webViews, webView => webView.name.toLowerCase());	
+	}
+	
+	public getWebView(platform: string, webViewName: string): IWebView {
+		let webViews = this.getWebViews(platform);
+		let webViewNameLowerCase = webViewName.toLowerCase();		
+		let webView = _.find(webViews, webView => webView.name.toLowerCase() === webViewNameLowerCase);
+		return webView;
+	}
+	
+	public enableWebView(platform: string, webViewName: string): IFuture<void> {
+		let webView = this.getWebView(platform, webViewName);
+		if(webView.default) { 
+			return this.enableDefaultWebView(platform);
+		} 
+		
+		return this.enableWebViewCore(webView);
+	}
+	
+	private enableWebViewCore(webView: IWebView): IFuture<void> {
+		return (() => {
+			if(!this.$pluginsService.isPluginInstalled(webView.marketplacePluginIdentifier)) {
+				this.$options.default = true;				
+				this.$pluginsService.configurePlugin(webView.marketplacePluginIdentifier).wait();
```

Comment:
`addPlugin` function has some checks and logic that we don't need in this case so I decided to skip them and use directly `configurePlugin` (`configurePlugin` is called internally from `addPlugin`)